.evergreen/run-kotlin-tests.sh

```diff
@@ -34,4 +34,4 @@ fi
 echo "Running Kotlin tests"
 
 ./gradlew -version
-./gradlew kotlinCheck -Dorg.mongodb.test.uri=${MONGODB_URI} ${MULTI_MONGOS_URI_SYSTEM_PROPERTY}
```

Comment:
Ah!this is probably why `kotlinCheck` was added as a task. Here nothing will run as there is no task passed to `gradlew`.  

A couple of options could be:

1) Keep `kotlinCheck` and only depend on test tasks
2) List all the kotlin projects and the tasks you want to run:

```
./gradlew bson-kotlin:test bson-kotlinx:test driver-kotlin-sync:test driver-kotlin-sync:integrationTest ...
```

Reply:
I think 2 is more explicit/readable about what we want to test 👍